data/colors/vars/component_dark.ts

```diff
@@ -146,5 +146,17 @@ export default {
   underlinenav: {
     icon: get('fg.subtle'),
     borderHover: get('neutral.muted')
+  },
+
+  actionListState: {
```

Comment:
IMO `actionListItem` is a little more clear than `actionListState`:

```suggestion
  actionListItem: {
```